### DISCUSSION TITLE
Porting to Android

### DISCUSSION THREAD

**Discussion by nachtlaubfrosch** [jendrikseipp/rednotebook#578] [Ideas]
Hi,
I love the workflow in rednotebook.
Even better with the last important Crossplattform "android".

And no, I can not Code unfortunately.

Are there any plans to do that?
KR
Big Fan

**Comment by jendrikseipp** [comment]
I currently don't have any plans to do this myself, but I'd be happy to assist someone else with the port.

**Comment by ortwic** [comment]
Same request from me here. I'd like to store thoughts and ideas on the go but I don't have a need for a summary.

I can code: TypeScript and C#.

My suggestion is sth. like that:

- [ ] WebApi for backend / CRUD operations on stored files in the cloud
  - [ ] Open question: how to store / access files securely from cloud?
    - Maybe Google Drive? [Here is a discussion ](https://answers.launchpad.net/rednotebook/+question/354365)
  - [ ] Parser for text files to return them into json format
- [ ] FrontEnd as PWA 
  - [ ] I'd choose Svelte as a Webframework
  - [ ] Propably Google Workspace Marketplace is the way to go instead of play store?

It would be much easier if files are not stored as proprietary text files but sth. like json or yml. 
Also for me there is no need for a summary. 

[Web folder](https://github.com/jendrikseipp/rednotebook/tree/master/web) is for summary only, right?

EDIT:
POC with sveltekit and google drive here: https://github.com/ortwic/red-notebook-pwa
